frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java

```diff
@@ -194,16 +193,21 @@ public void run(ProgressIndicator indicator) {
    * {@link GitMacheteGraphTableManager#refreshGraphTable()}.
    */
   @SuppressWarnings({"IllegalCatch"})
-  private void updateRepository(Path mainDirectoryPath, Path gitDirectoryPath, boolean isMacheteFilePresent) {
+  private void updateRepository(Path mainDirectoryPath, Path gitDirectoryPath, Path macheteFilePath,
+      boolean isMacheteFilePresent) {
     LOG.debug(() -> "Entering: mainDirectoryPath = ${mainDirectoryPath}, gitDirectoryPath = ${gitDirectoryPath}" +
         "isMacheteFilePresent = ${isMacheteFilePresent}");
+    IBranchLayoutManagerFactory branchLayoutManagerFactory = RuntimeBinding
```

Comment:
Make it a field of the manager to avoid reinstantiating every time `updateRepository` is called

Reply:
I am afraid that this is required. A user might change indent specs

Reply:
Ok... but then still, it's `branchLayoutManagerFactory.create(macheteFilePath, isMacheteFilePresent)` that's needed every time, not `instantiateSoleImplementingClass(IBranchLayoutManagerFactory.class)` IIUC

Reply:
You are right! :rocket: